### PR TITLE
[MIG] l10n_uy_currency_update: Migration to 18.0

### PR DESCRIPTION
**Pull request by mem-adhoc**

### CODE REVIEW

**Comment by roboadhoc** [comment]
[Pull request status dashboard](https://mergebot.adhoc.ar/ingadhoc/uruguay/pull/257).

File: l10n_uy_currency_update/__manifest__.py [hunk 1]
```diff
@@ -34,7 +34,7 @@
     'data': [
         'views/res_currency_views.xml',
```
Comment (zaoral) [inline l10n_uy_currency_update/__manifest__.py:34]: remove  'summary': '',

File: l10n_uy_currency_update/__manifest__.py [hunk 2]
```diff
@@ -34,7 +34,7 @@
...
     ],
-    'installable': False,
+    'installable': True,
     'auto_install': True,
     'application': False,
```
Comment (zaoral) [inline l10n_uy_currency_update/__manifest__.py:38]: remove store=False of field l10n_uy_have_bcu_code

File: l10n_uy_currency_update/__manifest__.py [hunk 3]
```diff
@@ -34,7 +34,7 @@
...
 }
```
Comment (zaoral) [inline l10n_uy_currency_update/__manifest__.py:39]: change variable rate with rates

Comment (zaoral) [inline l10n_uy_currency_update/__manifest__.py:39]: improve method _compute_l10n_uy_bcu_have_code,

- replace _parse_bcu_data with _get_bcu_currencies_mapping (Please check special case of UYI)
- you hwave a for and and if, you can simplify with two lines and a filtered

Comment (zaoral) [inline l10n_uy_currency_update/__manifest__.py:39]: Change method to do not compute all the rates. only the rate we want of the current currency

```python
    def action_l10n_uy_get_bcu_rate(self):
        self.ensure_one()
        date_api_client = Client(wsdl % 'awsultimocierre')
        last_closing_date = date_api_client.service.Execute()
        
        rate = self.env['res.company']._parse_bcu_data(self)
        if rate:
            raise UserError(_('Fecha Ultimo Cierre: %s\nRate: %s' % (rate[self.name][1], rate[self.name][0])))
        raise UserError(_('No se encontro cotizacion para esta Moneda'))
```

Comment (zaoral) [inline l10n_uy_currency_update/__manifest__.py:39]: Please all the lines max 120 chars

Comment (zaoral) [inline l10n_uy_currency_update/__manifest__.py:39]: Add logic to shod/hide buttons of the buttons in the res.currency form view

**Review by zaoral — CHANGES_REQUESTED** [review]

**Comment by zaoral** [comment]
@roboadhoc r+

**Pull request closed without merging**